libraries/plugins/es_objects/es_objects.cpp

```diff
@@ -403,6 +406,10 @@ void es_objects_plugin::plugin_startup()
    if(!graphene::utilities::checkES(es))
       FC_THROW_EXCEPTION(fc::exception, "ES database is not up in url ${url}", ("url", my->_es_objects_elasticsearch_url));
    ilog("elasticsearch OBJECTS: plugin_startup() begin");
+
+   const auto es_version = graphene::utilities::getVersion(es);
+   if(std::stoi(es_version.substr(0,1)) >= 7)
```

Comment:
This is wrong, will fail at ES10. Will change.

Reply:
should be better with https://github.com/bitshares/bitshares-core/pull/1997/commits/ae067f0999bd209444bde86dd3b8d481e10ff904